### ISSUE TITLE
Model page issue on CVAT 

### ISSUE THREAD

**Issue by himanshudw** [opencv/cvat#4866]
### My actions before raising this issue
Checked Could not get models from the server #2541
https://github.com/opencv/cvat/issues/4736
https://github.com/opencv/cvat/issues/2189

While doing auto annotation using nuclio it started giving error 
The nuclio desktop is showing exit status 1 and not showing any functions that we deploy using nuctl deploy. Nuclio is showing as orphan container though docker ps shows a port assigned to it.a Also nuctl get functions is giving error as below
/nuclio/pkg/cmdrunner/shellrunner.go:96
Failed to execute command: /bin/sh -c "/bin/cat /etc/nuclio/store/functions/nuclio/*"
/nuclio/pkg/platform/local/store.go:408
Failed to run cat command
/nuclio/pkg/platform/local/store.go:335
Failed to get functions
/nuclio/pkg/platform/local/store.go:253
Failed to read functions from local store
/nuclio/pkg/platform/local/store.go:217
mlusers@annotate-clone:~/cvat$
![image](https://user-images.githubusercontent.com/61216926/186903240-38b42fbe-e569-457f-b253-2f39e2fd0bb4.png)
![MicrosoftTeams-image (12)](https://user-images.githubusercontent.com/61216926/186903395-77ee5e97-1648-499b-bfb5-fb611ab95495.png)
![MicrosoftTeams-image (13)](https://user-images.githubusercontent.com/61216926/186903519-5783aa7c-6a86-4c97-825d-f13a8d8ee826.png)

### Expected Behaviour
The functions should be visible on Nuclio dashboard and model should be visible on CVAT under models section for auto annotation of new tasks  

### Current Behaviour
The model is not visible on the cvat models page and nuclio is showing exit status 1

### Your Environment
Docker version-20.10.3
Nuclio-nuctl-1.5.16-linux-amd64

Docker logs

[cvat_docker_logs.txt](https://github.com/opencv/cvat/files/9433197/cvat_docker_logs.txt)

**Comment by bsekachev** [comment]
@himanshudw

Please, work with nuclio repository first to resolve the issue.
If you are able to open nuclio dashboard and see your functions deployed there, but do not see them in CVAT, it is time to ask us :)

**Comment by himanshudw** [comment]
Thanks for your feedback and prompt reply..

**Comment by bsekachev** [comment]
I will close the issue, do not hesitate to reopen if you have any updates here :)

**Comment by himanshudw** [comment]
dfhsdhdz

**Comment by himanshudw** [comment]
#[bsekachev](https://github.com/bsekachev)- Can you please open the issue

**Comment by himanshudw** [comment]
I have made a new Linux VM and deployed the and nuctl from scratch but the new issue that I am facing issue with automatic annotation.

My docker ps gives me result as below
![image](https://user-images.githubusercontent.com/61216926/188465276-ff80d404-422e-4436-9116-442dc341e33e.png)

Now I have created a task with the images of dogs and have deployed the Faster RCNN and Yolo V3 for automatic annotation which healthy and displaying both on the Nuclio dashboard  and on the CVAT model's page.
![image](https://user-images.githubusercontent.com/61216926/188464049-ed8bfeec-4863-46e0-9dfc-1ed618902984.png)
![image](https://user-images.githubusercontent.com/61216926/188464231-f3c5f5cf-1e9d-4ea1-889b-1128b4d67a56.png)

But when I try to do automatic annotation using the same I get the output as below.
![image](https://user-images.githubusercontent.com/61216926/188464566-940749e4-39f6-4391-a7bf-83f4c49eb686.png)

Kindly help on this as the models are working well both on Nuclio and CVAT model page but not working when I use it on CVAT task.

Versions
CVAT Version- Dev
Nuctl Version- 1.10.4-linux-amd64
Docker Version- 20.10.17

**Comment by bsekachev** [comment]
Hi, @himanshudw 

Looks like you come across the same issue as here https://github.com/opencv/cvat/issues/4855

**Comment by bsekachev** [comment]
@himanshudw 

What is output of ``nuctl get functions`` for you?

**Comment by himanshudw** [comment]
Below is the output of the same

![image](https://user-images.githubusercontent.com/61216926/188544362-5564ee70-1756-4bbc-88bd-ca2832b9d269.png)

Also 1 more issue that I am facing is the that helloworld function has gotten unhealthy after the VM was restarted whereas the other two are still up and healthy. The same happened with all of the custom models that I deployed previously.

Below is the screenshot of the heeloworld function on the nuclio dashboard
![image](https://user-images.githubusercontent.com/61216926/188544884-b01feb74-6785-41ab-8e87-d4e7c8cf3738.png)

**Comment by bsekachev** [comment]
@azhavoro 
Do you have any ideas?

**Comment by azhavoro** [comment]
@himanshudw to fix `name or service not known` error please see https://github.com/opencv/cvat/pull/4907
Please address your question about `helloworld` function to nuclio repo.

**Comment by himanshudw** [comment]
I created a docker-compose.override.yml with the contents as below:

version: '3.3'

services:
  cvat_worker_low:
    extra_hosts:
      - "host.docker.internal:host-gateway"

Then I ran
docker compose -f docker-compose.yml \
    -f components/serverless/docker-compose.serverless.yml \
    -f docker-compose.override.yml up -d

Docker ps produces the below containers 
![image](https://user-images.githubusercontent.com/61216926/188616513-7abffa7e-fd53-454e-9636-120f1da26e54.png)

I have used 
nuctl deploy tf-faster-rcnn-inception-v2-coco-gpu \
--project-name cvat --path "serverless/tensorflow/faster_rcnn_inception_v2_coco/nuclio" --platform local \
--base-image tensorflow/tensorflow \
--desc "Faster RCNN from Tensorflow Object Detection GPU API" \
--image cvat/tf.faster_rcnn_inception_v2_coco_gpu \
 
to create the faster RCCN model in the nuclio dashboard but the error still remains as below

![image](https://user-images.githubusercontent.com/61216926/188617221-cebd2b0d-0e31-4b46-b3e5-da30ef643121.png)
 
Basically the error has changed from name or service not known to connection refused.

**Comment by azhavoro** [comment]
I don't see any function that is listening on port 49155 or 49156, try removing already deployed functions restart nuclio container and deploy again

**Comment by himanshudw** [comment]
I am really sorry I deleted the comment instead of editing it, yeah it worked out the way you have mentioned
